src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
Even though we're not supporting this layout on Android, it's possible for Android users to see this screen from a deeplink that has `firefox` in the URL (see https://github.com/mozilla/addons-frontend/issues/6136). I whipped up a simple responsive view (only took a minute) which I think is worth including:

```diff
diff --git a/src/amo/components/SecondaryHero/styles.scss b/src/amo/components/SecondaryHero/styles.scss
index 055cd935c..c35f98250 100644
--- a/src/amo/components/SecondaryHero/styles.scss
+++ b/src/amo/components/SecondaryHero/styles.scss
@@ -3,10 +3,16 @@
 .SecondaryHero {
   color: $black;
   display: grid;
-  grid-gap: 32px;
-  grid-template-columns: repeat(4, 1fr);
+  grid-gap: $padding-page;
+  grid-template-columns: auto;
   min-height: 216px;
-  padding: 48px 0 24px 0;
+  padding: $padding-page 0 0 0;
+
+  @include respond-to(large) {
+    grid-gap: 32px;
+    grid-template-columns: repeat(4, 1fr);
+    padding: 48px 0 24px 0;
+  }
 }
 
 .SecondaryHero-message {
@@ -23,7 +29,11 @@
 }
 
 .SecondaryHero-message-description {
-  padding: 8px 0 24px 0;
+  padding: $padding-page 0;
+
+  @include respond-to(large) {
+    padding: 8px 0 24px 0;
+  }
 }
 
 .SecondaryHero-module {
@@ -34,13 +44,23 @@
   flex-direction: column;
   font-size: $font-size-hero-description;
   line-height: 1.222;
-  padding: 0 24px 24px 24px;
+  padding: $padding-page;
+  padding-top: 0;
+
+  @include respond-to(large) {
+    padding: 24px;
+    padding-top: 0;
+  }
 }
 
 .SecondaryHero-module-icon {
   height: 60px;
-  margin: 30px auto 36px auto;
+  margin: $padding-page auto;
   width: 60px;
+
+  @include respond-to(large) {
+    margin: 30px auto 36px auto;
+  }
 }
 
 .SecondaryHero-module-link {
```

Reply:
Thanks! I agree that looks better on a small sceen!